src/main/resources/assets/api-docs.yaml

```diff
@@ -2959,110 +3032,149 @@ definitions:
-      non_tech_rus:
+        description: Describes the Research Use Statement
+      nonTechRus:
         type: string
-        description: 'Describes the Non Technical Summary.'
+        description: Describes the Non Technical Summary
       diseases:
         type: boolean
-        description: 'Defines if the purpose of the study is disease-related.'
+        description: Defines if the purpose of the study is disease-related
       methods:
         type: boolean
-        description: 'Defines if the purpose of the study is for Methods development.'
+        description: Defines if the purpose of the study is for Methods development
       controls:
         type: boolean
-        description: 'Defines if the purpose of the study is to increase the number of controls available for a comparison group.'
+        description: Defines if the purpose of the study is to increase the number of controls available for a comparison group
       population:
         type: boolean
-        description: 'Defines if the purpose of the study is to understand variation in the general population.'
+        description: Defines if the purpose of the study is to understand variation in the general population
       other:
         type: boolean
-        description: 'Defines if the purpose of the study is other than the above. If TRUE, triggers MANUAL REVIEW.'
+        description: Defines if the purpose of the study is other than the above
       otherText:
         type: string
-        description: 'Describes above field.'
+        description: Describes above field
       ontologies:
         type: array
         items:
           type: string
-        description: 'Array of Ontologies.'
+        description: Array of ontology terms
       forProfit:
         type: boolean
-        description: 'Describes if data will be used for commercial purposes.'
+        description: Describes if data will be used for commercial purposes
       oneGender:
         type: boolean
-        description: 'Describes if study is gender specific.'
+        description: Describes if study is gender specific
+      gender:
+        type: string
+        description: F/M if oneGender is specified
       pediatric:
         type: boolean
-        description: 'Describes if study is restricted to a pediatric population.'
-      illegalbehave:
+        description: Describes if study is restricted to a pediatric population
+      illegalbehavior:
         type: boolean
-        description: 'If TRUE, triggers MANUAL REVIEW.'
+        description: If TRUE, triggers MANUAL REVIEW.
       addiction:
         type: boolean
-        description: 'If TRUE, triggers MANUAL REVIEW.'
-      sexualdiseases:
+        description: If TRUE, triggers MANUAL REVIEW
+      sexualDiseases:
         type: boolean
-        description: 'If TRUE, triggers MANUAL REVIEW.'
-      stigmatizediseases:
+        description: If TRUE, triggers MANUAL REVIEW
+      stigmatizeDiseases:
```

Comment:
The variable was serialized as ```stigmatizedDiseases```, but it's written ```stigmatizeDiseases``` (missing the 'd' for the past-tense), should that be the case here?